src/collections/pages/fr/projects.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Projets
+permalink: fr/projets/
```

Comment:
You shouldn't need to specify this if `generatePermalink` is properly set up, let's discuss.